cmd/di.go

```diff
@@ -105,15 +106,14 @@ func (di *Dependencies) BootstrapServiceComponents(nodeOptions node.Options, ser
 
 	discoveryService := discovery.NewService(di.IdentityRegistry, di.IdentityRegistration, di.MysteriumClient, di.SignerFactory)
 
+	serviceOpenvpn := openvpn_service.NewService(nodeOptions, serviceOptions, di.IPResolver, di.LocationResolver)
```

Comment:
if the above is the "discoveryService" then this should also be "openvpnService"

Reply:
Renamed to `openvpnServiceManager`